app/components/works/edit/attached_file_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr class="attached-files <%= 'dz-complete' if uploaded? %>" data-dropzone-target="preview">
+    <%= form.fields_for :attached_files, attached_file do |file_form| %>
```

Comment:
Can you remind me what you learned about where a `<form>` element can live in a table? I assume/believe you tried having the form outside the `<tr>` and that didn't work?

FWIW, I don't think it matters much as long 1. browsers render it fine, and 2. it doesn't confuse screen readers.

Reply:
What I learned is that the location of the <%end%> tag determines where a hidden input tag is added by rails, and if it's added in the wrong spot, the browser gets confused.  The current code works and produces valid HTML even though the nesting of the <%end%> looks wrong.